src/frontend/js/widgets/SyllabusCourseRunsList/components/CourseProductItem/index.tsx

```diff
@@ -44,6 +44,16 @@ const messages = defineMessages({
     description: 'Course run languages',
     id: 'components.CourseProductItem.availableIn',
   },
+  noSeatsAvailable: {
+    defaultMessage: 'Sorry, no seats available for now',
+    description: 'Message displayed when no seats are available for the product',
+    id: 'components.CourseProductItem.noSeatsAvailable',
+  },
+  nbSeatsAvailable: {
+    defaultMessage: '{nb} seat(s) available',
```

Comment:
What do you think to display the remaining seats instead ? Furthermore, `react-intl` supports plural in messages.

```suggestion
    defaultMessage: `
count,
plural,
=0 {No seat}
one {1 seat}
other {# seats}
} remaining`,
```

You can take a look here : https://github.com/openfun/richie/blob/dcc209aaaee35a70f7a147a9159a9f305dc9c3f4/src/frontend/js/components/SaleTunnel/components/SaleTunnelStepValidation/CourseRunsList.tsx#L65